app/templates/components/forms/session-speaker-form.hbs

```diff
@@ -18,15 +18,15 @@
       {{t 'Choose from existing speaker(s)'}}
     </h4>
     <UiDropdown
-      @class="fluid multiple selection"
```

Comment:
we need to allow multiple speakers per session

Reply:
I'm done with the fixes. Please verify it once again.